p2p/providers/peerstore_provider/persistence/provider.go

```diff
@@ -0,0 +1,59 @@
+package persistence
```

Comment:
Please try to do `mv` in the future when possible. Seeing a file deleted and a new file created requires the reviewer to identify what the diff was or simply review all the business logic from scratch which takes a non-trivial amount of effort.

Reply:
:+1: I will make an effort to stage & commit file renames / moves independently of changes within the file; sometimes it's not obvious that such a change is upon you until it is. I think that's likely more representative of what happened in this case as I use my IDE to rename/move files and very rarely do `mv` or `git mv` in a terminal.